src/docs/asciidoc/antora.yml

```diff
@@ -0,0 +1,15 @@
+name: ROOT
+title: My Book
+version: ~
```

Comment:
since we are using the default generator with Antora 3, we can use `~` instead of `master`